Jenkinsfile

```diff
@@ -173,6 +175,7 @@ pipeline {
               expression { return INITIAL_COMMIT_PR == "true" }
               expression { return MERGE_CONDITIONS_SATISFIED == "true" }
               expression { return params.x86_64_macos }
+              expression { return params.nightly }
```

Comment:
I think this should be removed as we make a check for `params.iroha` above. We do not care whether it is nightly or a regular build